### PR TITLE
Fix several minor typo

### PR DESCRIPTION
**Pull request by sukey2008**

### CODE REVIEW

File: docs/installing-solidity.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ Docker
 ======
 
 Docker images of Solidity builds are available using the ``solc`` image from the ``ethereum`` organization.
-Use the ``stable`` tag for the latest released version, and ``nightly`` for potentially unstable changes in the develop branch.
+Use the ``stable`` tag for the latest released version, and ``nightly`` for potentially unstable changes in the development branch.
```
Comment (ekpyron) [inline docs/installing-solidity.rst]: This is not a typo - the branch is named ``develop``.

Comment (sukey2008) [reply docs/installing-solidity.rst]: Hi ekpyron, thanks for your review.
In this document, "development branch" was also mentioned above:  "they contain bleeding-edge code from the **development branch** and are not guaranteed to be always working."
I think it makes sense to make it consistent.
What do you think?

Comment (r0qs) [reply docs/installing-solidity.rst]: No, what he means is that our main branch is named `develop` and that sentence mentions this branch. 

So I would suggest it to:
```suggestion
Use the ``stable`` tag for the latest released version, and ``nightly`` for potentially unstable changes in the ``develop`` branch.
```

Comment (sukey2008) [reply docs/installing-solidity.rst]: @r0qs: Thanks for your suggestion, this is really helpful!
I've modified now the sentence according to your suggestion.

**Review by mehtavishwa30 — APPROVED** [review]

**Pull request merged**